util-core/src/main/scala/com/twitter/util/Time.scala

```diff
@@ -195,19 +195,29 @@ trait TimeLike[This <: TimeLike[This]] extends Ordered[This] { self: This =>
   /** The difference between the two `TimeLike`s */
   def diff(that: This): Duration
 
+  private[this] def round(f: (Long, Long) => Long, x: Duration): This = (this, x) match {
```

Comment:
Done (I copied increment, not sure it's the best, but I didn't have anything better).